src/theme/variables.js

```diff
@@ -52,9 +54,11 @@ module.exports = {
   mediumFontSize: '14px',
   orange500: '#e8663d',
   primary,
-  primary200: '#EFFAF8',
+  primaryLite,
```

Comment:
Why export these if they are under `primaryN`?

Reply:
`primary200`, `primary300` don't make any sense tbh. Names should be descriptive, or at least mnemonic.

Reply:
Don't match the design colour pallette?

Reply:
Matching Figma is a good point. We should align on this with the designers.